common/utils-java/src/main/java/org/apache/spark/network/util/JavaUtils.java

```diff
@@ -737,4 +737,34 @@ public static boolean isTesting() {
   public static boolean isUnix = Stream.of("AIX", "HP-UX", "Irix", "Linux", "Mac OS X", "Solaris",
     "SunOS", "FreeBSD", "OpenBSD", "NetBSD")
     .anyMatch(prefix -> osName.regionMatches(true, 0, prefix, 0, prefix.length()));
+
+  /**
+   * Throws IllegalArgumentException if the given object is null.
+   * Keep this clone of CommandBuilderUtils.checkNotNull synced with the original.
+   */
+  public static void checkNotNull(Object o, String arg) {
```

Comment:
In this regard, we can directly use java.util.Objects#requireNonNull.

Reply:
Let me remove this.